blog/_posts/2016-10-12-Hackbright.md

```diff
@@ -0,0 +1,123 @@
+---
+ title:
+ layout: post
+ created_at: Fri Oct 12 2016
+ permalink: blog/2016-09-30-Hackbrighter-End-of-Summer
+ current: blog
+ author: Melissa Fabros and Patricia Arbona
+ twitter: HeartRGSoC_sf
+ categories:
+ - student-posts
+ ---
+ # Team Hackbrighter's Amazing Summer of Code Experience
```

Comment:
The title will be shown automatically if you add it to the title in the "frontmatter" (metadata), see comment above. Can you remove it from here and add it there?